content/post/stats_at_work.md

```diff
@@ -7,89 +7,72 @@ date: 2018-07-26T20:35:46-04:00
-MA: Sure but what are the chances of that?
+Diane: Right. But how do you know that B wouldn't have caught up to A eventually?  
+Marty: Sure but what are the chances of that?
 
-DS: Yeah good question. We run tests to be able to answer questions like that - "What are the chances that even though A is currently better than B, that this trend continues"?  
-MA: I see. And how exactly do we answer that?
+Diane: Yeah good question. We run tests to be able to answer questions like that - "What are the chances that even though A is currently better than B, this trend continues"?  
+Marty: I see. And how exactly do we answer that?
 
-DS: Well, we usually measure how confident we are that A is better than B using something called the p-va-  
-MA: Yeah p-value. I know about that. If the p-value is less than 5% then the test is significant otherwise it's not.
+Diane: Well, we usually measure how confident we are that A is better than B using something called the p-va-  
+Marty: Yeah p-value. I know about that. If the p-value is less than 5% then the test is significant otherwise it's not.
 
-DS: Yeah more or less.  
-MA: OK. And I know that the p-value is not the probability of A being better than B. It's something else...but what is it exactly?
+Diane: Yeah more or less.  
+Marty: Okay. But, uh, honestly I don't know what that means, I just remember reading it over and over again.
 
-DS: The exact definition of the p-value is not super helpful, but it basically quantifies the probability of seeing what we saw - namely that A is better than B - if in fact A was the same as B! In other words, if A and B were equal, then how unlikely was our result?   
-MA: So a small p-value means that if A and B were equal, then our result would be highly unlikely. And this leads us to conclude that the result of the test can be trusted? It's accurate?
+Diane: Listen this stuff is very confusing so don't feel bad. The exact definition of the p-value is not super helpful, but it basically quantifies the probability of seeing what we saw - namely that A is better than B - if in fact A was the same as B! In other words, if A and B were equal, then how unlikely was our result?   
+Marty: So a small p-value means that if A and B were equal, then our result would be highly unlikely.
 
-DS: Kinda yeah. To be clear, there are different ways to quantify this kind of stuff. This type of analysis, called hypothesis testing, is the most common one used by the vast majority of scientists.   
-MA: Okay. So at the end of the test, you calculate the p-value and saw that it was under 5%. 
+Diane: Right. To tie in the 5%, this means that if A and B had the same conversion rate, then our test would say there was a difference 5% of the time. Imagine running this test 20 times. In one of those 20 tests we might incorrectly say that A is better than B. This is why the p-value is also known as the probability of making a false positive.   
```

Comment:
> that if A and B had the same conversion rate,

to

> that if A and B had the same "true" conversion rate,

and

> would say there was a difference 5% of the time

to

> would *incorrectly* say there was a difference 5% of the time.

for emphasis / clarity

Reply:
It might also be worth adding a bit of dialogue about what "true conversion rate" is getting at. I explain this by saying something like: we want to know what the impact of this change will be if we release it and let it run forever for all of our customers. This "forever for all" conversion rate is the "true" conversion rate that we can't know, but that we're trying to get signal for using the test. P-values help us understand how clear the signal is when we try to estimate this value.

... or something like that.

Reply:
Will try to clarify what I mean by "true" conversion rate. It kind of touches upon the whole point of statistics (i.e., to approximate population parameters with samples) but maybe a brief sentence or two is sufficient.